documentation/modules/oauth/con-oauth-authentication-client-options.adoc

```diff
@@ -3,33 +3,36 @@
 // assembly-oauth-authentication.adoc
 
 [id='con-oauth-authentication-client-options-{context}']
-= OAuth 2.0 client authentication flow
+= OAuth 2.0 client authentication flows
 
-In this section, we explain and visualize the communication flow between Kafka client, Kafka broker, and authorization server during Kafka session initiation.
-The flow depends on the client and server configuration.
+[role="_abstract"]
+OAuth 2.0 authentication flows depend on the underlying Kafka client and Kafka broker configuration.
+The flows must also be supported by the authorization server used.
 
-When a Kafka client sends an access token as credentials to a Kafka broker, the token needs to be validated.
+Kafka broker listener configuration determines how clients authenticate using an access token.
+The client can pass a client ID and secret to request an access token.
 
-Depending on the authorization server used, and the configuration options available, you may prefer to use:
+If a listener is configured to use PLAIN authentication, the client can authenticate with a client ID and secret or username and access token.
+These values are passed as the `username` and `password` properties of the PLAIN mechanism.
 
-* Fast local token validation based on JWT signature checking and local token introspection, without contacting the authorization server
-* An OAuth 2.0 introspection endpoint provided by the authorization server
+Listener configuration supports the following token validation options:
 
-Using fast local token validation requires the authorization server to provide a JWKS endpoint with public certificates that are used to validate signatures on the tokens.
+* Fast local token validation based on JWT signature checking and local token introspection, without contacting an authorization server.
```

Comment:
Refrain from using fragments.

Maybe try...

You can use fast local token validation based on JWT signature checking and local token introspection, without contacting an authorization server.